airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/load/data/UnionTypeToDisjointRecord.kt

```diff
@@ -8,30 +8,24 @@ import io.airbyte.cdk.load.message.DestinationRecord
 
 class UnionTypeToDisjointRecord : AirbyteSchemaIdentityMapper {
     override fun mapUnion(schema: UnionType): AirbyteType {
-        val (nullOptions, nonNullOptions) = schema.options.partition { it is NullType }
-        if (nonNullOptions.size < 2) {
+        if (schema.options.size < 2) {
```

Comment:
unrelated thought:
* if size == 1 -> should we just return `schema.options.first()`?
* if size == 0 -> wtf :P (uh, maybe return UnknownType? `{oneOf: []}` is a pretty stupid schema)

Reply:
In theory nothing should get to this point with a singleton Union

actually singleton unions shouldn't be representable at all, ideally

like ideally the type classes would be set up in a way where you're always getting an `AirbyteType` when you try to create a `Union`, and
* singletons just become the type
* redundancies disappear into `Set<Options>`

Reply:
Which means that this guard is probably a little vestigial. :) It only made sense when there were null types.

I'll remove it in a cleanup pass.